packages/@divvi/mobile/src/send/SendEnterAmount.tsx

```diff
@@ -76,13 +77,14 @@ function SendEnterAmount({ route }: Props) {
     })
   }
 
+  const prepareTransactions = usePrepareSendTransactions()
   const {
     prepareTransactionsResult,
     refreshPreparedTransactions,
     clearPreparedTransactions,
     prepareTransactionError,
     prepareTransactionLoading,
-  } = usePrepareSendTransactions()
+  } = prepareTransactions
```

Comment:
do we need to pass everything to the next screen? we don't allow proceeding unless the transaction is possible, so why not pass only `prepareTransactionsResult`? we may also want to serialise the prepared transaction (i vaguely remember that is recommended with this navigation lib)